Anonymiser.py

```diff
@@ -0,0 +1,58 @@
+import os
+import re
+import hashlib
+from os.path import join, getsize, isfile
+
```

Comment:
This is your library of regex for anonymisation.  You need to mark it more clearly.  Is there any other logic that you are using for this?  If not make that clear that this is the whole thing.  What is the order of precedence in the processing?